hazelcast/src/test/java/com/hazelcast/query/impl/getters/MethodGetterTest.java

```diff
@@ -97,6 +97,17 @@ public void constructor_whenModifierIsPositionAndMethodReturnTypeIsCollection_th
         new MethodGetter(null, limbCollectionMethod, "[0]", null);
     }
 
+    @Test
+    public void getValue_whenFieldIsEnum()
```

Comment:
Can you explain the motivation for this test? It is not failing with current `master` and it does not test `MethodGetter` with enum elements, but maybe I 'm missing something?